### ISSUE TITLE
HTML table overlap on page: https://www.php.net/manual/en/resource.php

### ISSUE THREAD

**Issue by Andro-H** [php/doc-en#792]
From manual page: https://www.php.net/manual/en/resource.php

on mozilla and chrome it looks like this:

![Screenshot 2021-07-16 at 16 41 32](https://user-images.githubusercontent.com/87533496/125958307-0edf4b1d-8b5b-4059-b6bc-049fd1e52e0c.png)

**Comment by cmb69** [comment]
Yes, that is a known issue; I'm pretty sure that there is an open ticket on https://bugs.php.net, but I can't find it. In my opinon, this shouldn't be a table, but rather a (nested) `<variablelist>`.

**Comment by zors1** [comment]
The quickest solution would probably be adding `word-break: break-word;` to each `<td>` element and removing empty columns from the "interbase blob" row (it has 3 extra columns for some reason).

**Comment by jabir-sabitov** [comment]
The problem is still relevant.

![image](https://github.com/user-attachments/assets/b6b42375-72ed-4838-bece-4da5fc7f6d76)

**Comment by cmb69** [comment]
We have that problem on other pages as well, e.g.

![filter](https://github.com/user-attachments/assets/7ae7de81-8bb7-4632-b40a-0f7461ae9582)

**Comment by Girgias** [comment]
> We have that problem on other pages as well, e.g.

Right, but those pages need to use something else then tables. As the column width are dictated by the widest single element in a column.

**Comment by jimwins** [comment]
Changing these wide tables to a different structure would be great, but a short-term solution could be adding a CSS rule to web-php to cause the overflow to scroll instead of overlap:

```php
#layout-content {
  overflow: scroll;
}```

(Could be added to the block at line 14 of `styles/theme-medium.css`.)

**Comment by vrana** [comment]
It renders correct for me now. Could this be closed?

![Image](https://github.com/user-attachments/assets/f9f03b68-5802-4589-bb88-26cfcc8a0bde)

**Comment by Andro-H** [comment]
Thanks guys,
All good indeed, closing it as completed.
